torchmultimodal/diffusion_labs/models/adm_unet/adm.py

```diff
@@ -453,7 +479,7 @@ def adm_unet(
     time_embed_dim: int = 512,
     cond_embed_dim: int = 2048,
     clip_embed_dim: int = 768,
-    clip_embed_name: str = "clip_image",
+    clip_embed_name: str = DEFAULT_EMBED_NAME,
     predict_variance_value: bool = True,
     # ADMUNet args
     image_channels: int = 4,
```

Comment:
Can't comment lower than this line. But on line 490 I think this should be: constructs a conditional ADM U-Net